docker-compose.yml

```diff
@@ -17,13 +17,13 @@ services:
       AUDIENCE: "kartoffel"
```

Comment:
what about the new env.var `GITHUB_BRANCH`?

Reply:
not in use (I used bash).